app.json

```diff
@@ -60,13 +60,13 @@
       [
         "expo-camera",
         {
-          "microphonePermission": false
+          "microphonePermission": "Allow $(PRODUCT_NAME) to access your microphone"
         }
       ],
       [
         "expo-image-picker",
         {
-          "microphonePermission": false
+          "microphonePermission": "Allow $(PRODUCT_NAME) to access your microphone"
         }
```

Comment:
Because for iOS, we encounter the error I shared above. It says that `NSCameraUsageDescription` and `NSMicrophoneUsageDescription` should be in `Info.plist`. Otherwise the app crashes. 🤷‍♂️

Reply:
isn't this the part of

```
 "infoPlist": {
        "NSCameraUsageDescription": "Diese App nutzt die Kamera um QR-Codes zu scannen.",
        ...
```

could we also add `NSMicrophoneUsageDescription` there and let them false here in the package usage?
if not, please consider German texts.

Reply:
I tried this but if `microphonePermission` is false, `NSMicrophoneUsageDescription` is not added to `Info.plist`.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/41820c7808bf23ddbd16de33515e10c0ab70b07f